src/data-accessor.ts

```diff
@@ -22,15 +23,15 @@ function parseCsvToPower(csv: CsvFile): Power[] {
     return {
       name: row[0] as string,
```

Comment:
Question: Is all this type casting necessary or is there potentially a cleaner way?